internal/app/connectconformance/test_case_library_test.go

```diff
@@ -901,6 +901,44 @@ func TestPopulateExpectedResponse(t *testing.T) {
 	})
 	require.NoError(t, err)
 
+	idempotentUnaryErrorJSONReqInfo, err := anypb.New(&conformancev1.ConformancePayload_RequestInfo{
```

Comment:
The better way is to use JSON here. The struct literal syntax is much more verbose (especially since it requires lots of error-checking code to construct `Any` messages).

See `TestExpandRequestData` for an example.

Reply:
I created a TODO for this to refactor this if that's cool. I think I'd rather do that as part of its own PR.

Won't converting to JSON still lead to a bunch of repeated text since I can't (easily) reuse strings for say the headers/trailers, etc.? Not sure if it would be a wash with verbosity.

Reply:
> I created a TODO for this to refactor this if that's cool. 

Most definitely. 👍